application/helpers/LayoutHelper.php

```diff
@@ -338,6 +362,11 @@ public function getAdminFooter(string $url, bool $return = false, bool $question
 
         $aData['imageurl'] = Yii::app()->getConfig("imageurl");
         $aData['url'] = $url;
+
+        $oEvent = new PluginEvent('beforeFooterRender');
```

Comment:
Not part of this PR

Reply:
No :the event can be in this commit, but the content added/returned/updated in the plugin : no ?

Reply:
Yea but only if the event is needed. Not sure yet.

Reply:
> Yea but only if the event is needed. Not sure yet.

it can be just great to have a GUI event here

Reply:
beforeCloseHtml is only for survey rendering then, not admin view?

Reply:
> beforeCloseHtml is only for survey rendering then, not admin view?

[Yes](https://github.com/LimeSurvey/LimeSurvey/blob/8b2d1c7ac40b40caea6a90df2df581fe6bb11661/application/core/LSYii_ClientScript.php#L620), and not allowed to add content at any place, only at end.

I only use it to add script sometimes.